internal/peer/lifecycle/chaincode/chaincode.go

```diff
@@ -123,7 +124,7 @@ func ResetFlags() {
 	flags.DurationVar(&waitForEventTimeout, "waitForEventTimeout", 30*time.Second,
 		fmt.Sprint("Time to wait for the event from each peer's deliver filtered service signifying that the 'invoke' transaction has been committed successfully"))
 	flags.StringVarP(&packageID, "package-id", "", "", "The identifier of the chaincode install package")
-	flags.IntVarP(&sequence, "sequence", "", 1, "The sequence number of the chaincode definition for the channel")
+	flags.IntVarP(&sequence, "sequence", "", 0, "The sequence number of the chaincode definition for the channel")
```

Comment:
This is a change in the default value for this command.  I think I prefer this default, as it works with the defaults provided by the system chaincode (0 is 'unset').  But, @nikhil550 @denyeart will this break anything in our samples/docs etc.?

Reply:
None of the samples use the default value for the sequence parameter.

Reply:
I agree with the default change, we'll need to release note it with v2.2.
However I would expect the default change to come in a separate PR, and also the command line help text that mentions the default needs to be updated:

`--sequence int                   The sequence number of the chaincode definition for the channel (default 1)`

Reply:
@jyellick @nikhil550 @denyeart
Thank you for your reviews.

Based on the comments, I submitted a separate PR for this as below.
The PR also includes updates to the help docs.
https://github.com/hyperledger/fabric/pull/1262

Immediately after the separate PR has been merged, I will resubmit the updated patch to address the other comments.